src/components/ha-markdown.ts

```diff
@@ -53,19 +68,46 @@ export class HaMarkdown extends LitElement {
       margin: var(--ha-space-1) 0;
     }
     a {
-      color: var(--primary-color);
+      color: var(--markdown-link-color, var(--primary-color));
     }
     img {
+      background-color: rgba(10, 10, 10, 0.15);
+      border-radius: var(--markdown-image-border-radius);
       max-width: 100%;
+      min-height: 2lh;
```

Comment:
This is making images bigger than they should be. 

From a branch I have:

<img width="2096" height="660" alt="Image" src="https://github.com/user-attachments/assets/a8c670e4-4951-4a86-bb22-0d63cedabe09" />